.github/workflows/nginx-otel-module-check.yml

```diff
@@ -81,17 +61,21 @@ jobs:
         run: |
           auto/configure --with-compat --with-debug --with-http_ssl_module \
                          --with-http_v2_module --with-http_v3_module
-          make -j 4
-      - name: Checkout lib from nginx-tests
-        uses: actions/checkout@v4
+          make -j $(nproc)
+      - name: Download otelcol
+        uses: robinraju/release-downloader@v1.11
         with:
-          repository: nginx/nginx-tests
-          sparse-checkout: |
-            lib
-          path: nginx-tests
+          repository: open-telemetry/opentelemetry-collector-releases
+          tag: v0.113.0
+          fileName: otelcol_0.113.0_linux_amd64.tar.gz
+          extract: true
       - name: Run tests
         working-directory: tests
+        env:
+          TEST_NGINX_GLOBALS: |
+            load_module ${{ github.workspace }}/build/ngx_otel_module.so;
```

Comment:
OK, I don't mind having it as a relative path.

Reply:
This was just en example, although, I'd like to be able to use relative path when running tests locally.

Reply:
The relative path is impossible because we use a testdir as root directory for nginx. But I have moved the TEST_NGINX_GLOBALS definition to the command string and have used shell vars to point to the module.

Reply:
The relative path is secondary (although useful) property here. The main ask is to provide module specific convenience option. There is no need to rely on environment variables when there is [pytest_addoption](https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.hookspec.pytest_addoption).

Reply:
If you look at nginx_config fixture you will see that I use TEST_NGINX_* variables to get some custom config options during run time. This is the main idea of using env vars.

Reply:
Yeah, and I'm saying that pytest provides a better way to configure test parameters.

And I don't think we need any "TEST_NGINX_GLOBALS_*" for these tests, just a way to specify module path.

Reply:
Switched to pytest_addoption.